tests/menu/accelentry.cpp

```diff
@@ -20,73 +20,81 @@
 #endif // WX_PRECOMP
 
 #include "wx/accel.h"
-#include "wx/scopedptr.h"
-
-class AccelEntryTestCase : public CppUnit::TestCase
-{
-public:
-    AccelEntryTestCase() {}
-
-private:
-    CPPUNIT_TEST_SUITE( AccelEntryTestCase );
-        CPPUNIT_TEST( Create );
-        CPPUNIT_TEST( ToFromString );
-    CPPUNIT_TEST_SUITE_END();
-
-    void Create();
-    void ToFromString();
-
-    wxDECLARE_NO_COPY_CLASS(AccelEntryTestCase);
-};
-
-// register in the unnamed registry so that these tests are run by default
-CPPUNIT_TEST_SUITE_REGISTRATION( AccelEntryTestCase );
-
-// also include in its own registry so that these tests can be run alone
-CPPUNIT_TEST_SUITE_NAMED_REGISTRATION( AccelEntryTestCase, "AccelEntryTestCase" );
 
 namespace
 {
 
 void CheckAccelEntry(const wxAcceleratorEntry& accel, int keycode, int flags)
 {
-    CPPUNIT_ASSERT_EQUAL( keycode, accel.GetKeyCode() );
-    CPPUNIT_ASSERT_EQUAL( flags, accel.GetFlags() );
+    REQUIRE( keycode == accel.GetKeyCode() );
```

Comment:
This is pretty minor, but I think these 2 `REQUIRE`s could, and hence should, actually be `CHECK()`s.

Reply:
All the REQUIRE() are now CHECK()